sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.net8.0.cs

```diff
@@ -2687,4 +2767,31 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UpdateNetworkSiblingSetContent>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UpdateNetworkSiblingSetContent>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class UsageName : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.NetApp.Models.UsageName>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UsageName>
+    {
+        internal UsageName() { }
+        public string LocalizedValue { get { throw null; } }
+        public string Value { get { throw null; } }
+        protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.NetApp.Models.UsageName System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.NetApp.Models.UsageName>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.NetApp.Models.UsageName>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.NetApp.Models.UsageName System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UsageName>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UsageName>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UsageName>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class UsageResult : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.NetApp.Models.UsageResult>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UsageResult>
```

Comment:
I think this should be `NetAppUsageResult`

Reply:
UsageResult renamed.